lib/services/coins/epiccash/epiccash_wallet.dart

```diff
@@ -251,7 +251,16 @@ Future<String> deleteEpicWallet({
 
   final wallet = await secureStore.read(key: '${walletId}_wallet');
 
-  return compute(_deleteWalletWrapper, wallet!);
+  if (wallet == null) {
+    return "Tried to delete non existent epic wallet file with walletId=$walletId";
+  } else {
+    try {
+      return _deleteWalletWrapper(Tuple2(wallet, config!));
+    } catch (e, s) {
+      Logging.instance.log("$e\n$s", level: LogLevel.Error);
+      return "deleteEpicWallet($walletId) failed...";
```

Comment:
It would be nice here to return a response object that can pass an error instead of needing to check the Future<String> returned